distribution/sender/routes/sender-activemq-publisher.xml

```diff
@@ -0,0 +1,146 @@
+                                    #{body.metadata.setDateSent(T(java.time.LocalDateTime).now())}
+                                    #{body.metadata.setOperation(getProperty('senderSyncMessage').operation)}
+                                    #{body.metadata.setRequestUuid(getProperty('senderSyncMessage').requestUuid)}
+                                    #{body.metadata.setMessageUuid(getProperty('senderSyncMessage').messageUuid)}
+                                    #{body.metadata.setSnapshot(getProperty('senderSyncMessage').snapshot)}
+                                </spel>
+                            </script>
+                            <setBody>
+                                <method beanType="org.openmrs.eip.component.utils.JsonUtils" method="marshall(${body})" />
+                            </setBody>
+            
+                            <log loggingLevel="DEBUG" message="Sync payload -> ${body}" />
+    
+                            <when>
+                                <simple>{{openmrs.eip.dbsync.encryption.enabled}} == true</simple>
+                                <log message="Encrypting entity payload.." />
+            
+                                <process ref="pgpEncryptService" />
+            
+                                <log message="Encrypted entity payload -> ${body}" loggingLevel="TRACE" />
+                            </when>
+    
+                            <log message="Sending entity to sync destination: {{camel.output.endpoint}}" />
+            
+                            <toD uri="{{camel.output.endpoint}}" />
+            
+                            <log message="Entity sent!" loggingLevel="DEBUG" />
+                    
+                            <script>
+                                <spel>
+                                    #{getProperty('senderSyncMessage').markAsSent()}
+                                </spel>
+                            </script>
+                            
+                            <setBody>
+                                <simple>${exchangeProperty.senderSyncMessage}</simple>
+                            </setBody>
+                            
+                            <log message="Updating sender sync message with identifier ${exchangeProperty.senderSyncMessage.identifier} to ${exchangeProperty.senderSyncMessage.status}" loggingLevel="DEBUG" />
+                            
+                            <to uri="jpa:SenderSyncMessage" />
+            
+                            <log message="Successfully updated sender sync message with identifier ${exchangeProperty.senderSyncMessage.identifier} to ${exchangeProperty.senderSyncMessage.status}" />
+                        </when>
+                        
+                        <otherwise>
+                            <log message="No entity found in the database matching identifier ${exchangeProperty.senderSyncMessage.identifier}" loggingLevel="DEBUG" />
```

Comment:
I think this should explicitly say **Requested entity not found in the database matching identifier ${exchangeProperty.senderSyncMessage.identifier} in table ${exchangeProperty.senderSyncMessage.tableName}**

Reply:
And you need to log the message at info level like before, not debug

Reply:
Alright. Done.

Reply:
During our call it was decided to change this message to: 
`No entity found in the database matching identifier ${exchangeProperty.senderSyncMessage.identifier} in table ${exchangeProperty.senderSyncMessage.tableName}`